src/lib/import-export/export/export-database.ts

```diff
@@ -47,15 +47,25 @@ export async function exportDatabaseToMultipleFiles(
 	}
 
 	const tablesResult = await server.executeWpCliCommand(
-		`sqlite tables --format=csv --require=/tmp/sqlite-command/command.php`
+		`sqlite tables --format=json --require=/tmp/sqlite-command/command.php`
```

Comment:
Switching to JSON, we can easily validate the output. If HTML or text content leaks into the command output, it will always produce invalid JSON.